components/charts/pieChart.js

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+import { db } from "../../firebaseConfig";
+import { Pie } from "react-chartjs-2";
+
+export default function PieChart(props) {
+  const [data, setData] = useState({
+    val: [],
+    label: [],
+    color: [],
+  });
+
+  db.collection("1A Data")
```

Comment:
same comment as other one about logic in diff file.

Reply:
same response as other